<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="currentRow">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>User Interface</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Behavior</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thumbnail</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Volume</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Advanced</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="uiPage">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Icons</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Size of big icons:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="bigIconSize"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Size of small icons:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="smallIconSize"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Size of thumbnails:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="thumbnailIconSize"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Size of side pane icons:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="sidePaneIconSize"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="iconThemeLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Icon theme:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="iconTheme">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Window</string>
           </property>
           <layout class="QFormLayout" name="formLayout_7">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <item row="0" column="0" colspan="2">
             <widget class="QCheckBox" name="alwaysShowTabs">
              <property name="text">
               <string>Always show the tab bar</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QCheckBox" name="showTabClose">
              <property name="text">
               <string>Show 'Close' buttons on tabs	</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QCheckBox" name="rememberWindowSize">
              <property name="text">
               <string>Remember the size of the last closed window</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Default width of new windows:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Default height of new windows:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="fixedWindowHeight">
              <property name="maximum">
               <number>32768</number>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="fixedWindowWidth">
              <property name="maximum">
               <number>32768</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="behaviorPage">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Browsing</string>
           </property>
           <layout class="QFormLayout" name="formLayout_5">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0" colspan="2">
             <widget class="QCheckBox" name="singleClick">
              <property name="text">
               <string>Open files with single click</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Delay of auto-selection in single click mode (0 to disable)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Default view mode:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="viewMode"/>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="autoSelectionDelay">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string> sec</string>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>File Operations</string>
           </property>
           <layout class="QFormLayout" name="formLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Confirm before:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QFrame" name="frame">
              <layout class="QFormLayout" name="formLayout_8">
               <item row="1" column="0">
                <widget class="QCheckBox" name="configmTrash">
                 <property name="text">
                  <string>Moving files or folders to the trash</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QCheckBox" name="configmDelete">
                 <property name="text">
                  <string>Deleting files or folders</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="useTrash">
              <property name="text">
               <string>Move deleted files to &quot;trash bin&quot; instead of erasing from disk.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="thumbnailPage">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="showThumbnails">
           <property name="text">
            <string>Show thumbnails of files</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="thumbnailLocal">
           <property name="text">
            <string>Only show thumbnails for local files</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Do not generate thumbnails for image files exceeding this size:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="maxThumbnailFileSize">
           <property name="suffix">
            <string> KB</string>
           </property>
           <property name="maximum">
            <number>1048576</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="volumePage">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Auto Mount</string>
           </property>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QCheckBox" name="mountOnStartup">
              <property name="text">
               <string>Mount mountable volumes automatically on program startup</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="mountRemovable">
              <property name="text">
               <string>Mount removable media automatically when they are inserted</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="autoRun">
              <property name="text">
               <string>Show available options for removable media when they are inserted</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>When removable medium unmounted:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QRadioButton" name="closeOnUnmount">
              <property name="text">
               <string>Close tab containing removable medium</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="goHomeOnUnmount">
              <property name="text">
               <string>Change folder in the tab to home folder</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="advancedPage">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Programs</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Terminal emulator:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Switch user command:</string>
              </property>
              <property name="buddy">
               <cstring>suCommand</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="suCommand"/>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Examples: &quot;xterm -e %s&quot; for terminal or &quot;gksu %s&quot; for switching user.
%s = the command line you want to execute with terminal or su.</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Archiver integration:</string>
              </property>
              <property name="buddy">
               <cstring>archiver</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="archiver"/>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="terminal">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="siUnit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Use SI decimal prefixes instead of IEC binary prefixes</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>smallIconSize</tabstop>
  <tabstop>bigIconSize</tabstop>
  <tabstop>thumbnailIconSize</tabstop>
  <tabstop>sidePaneIconSize</tabstop>
  <tabstop>iconTheme</tabstop>
  <tabstop>alwaysShowTabs</tabstop>
  <tabstop>showTabClose</tabstop>
  <tabstop>singleClick</tabstop>
  <tabstop>viewMode</tabstop>
  <tabstop>configmDelete</tabstop>
  <tabstop>useTrash</tabstop>
  <tabstop>showThumbnails</tabstop>
  <tabstop>mountOnStartup</tabstop>
  <tabstop>suCommand</tabstop>
  <tabstop>archiver</tabstop>
  <tabstop>siUnit</tabstop>
  <tabstop>autoRun</tabstop>
  <tabstop>mountRemovable</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>thumbnailLocal</tabstop>
  <tabstop>maxThumbnailFileSize</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleClick</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rememberWindowSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>fixedWindowWidth</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleClick</sender>
   <signal>toggled(bool)</signal>
   <receiver>autoSelectionDelay</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rememberWindowSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>fixedWindowHeight</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rememberWindowSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_12</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rememberWindowSize</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_13</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
